### PR TITLE
readme expanded and formatted

### PR DESCRIPTION
**Pull request by rnewbigging**
## Context
 

SAP/cloud-sdk-java-backlog#345

Expanded the ReadMe for legibility and additional resources, including use cases, documentation links, and table of contents.

### Feature scope:

 
- [x] Name and description
- [x] Table of contents
- [x] Version 5 migration
- [x] Installation steps
- [x] Use Cases
    - [x] OData
    - [x] OpenAPI
    - [x] Connectivity
- [x] Release Notes link
- [x] Support
- [x] Contribution
- [x] Documentation link
- [x] Tutorial link 

## Definition of Done

- [x] Functionality scope stated & covered
- [ ] ~Tests cover the scope above~
- [ ] ~Error handling created / updated & covered by the tests above~
- [ ] ~Documentation updated~
- [ ] ~Release notes updated~

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,11 +10,32 @@
...
 The SAP Cloud SDK for Java makes it easy to build highly connected, resilient, multi tenant SaaS applications on the SAP Business Technology Platform (SAP BTP).
 It enables you to perform outbound requests using various protocols while only writing the business logic.
 
+## Release Notes
+- For an overview of the latest change, please refer to our [release notes.](https://sap.github.io/cloud-sdk/docs/java/v5/release-notes).
+
+## Documentation
+- Our documentation can be found [here](https://sap.github.io/cloud-sdk/docs/java/getting-started), which includes an [overview of available tutorials](https://sap.github.io/cloud-sdk/docs/java/guides/tutorial-overview-sdk-java).
+
 ## Requirements and Setup
```
Comment (rnewbigging) [inline README.md:39]: Is the given link to the "getting started" documentation sufficient or do we want this expanded?

File: README.md [hunk 2]
```diff
@@ -35,6 +56,41 @@ mvn clean install
 ```
 To skip tests, add `-Dmaven.test.skip`.
```
Comment (rnewbigging) [inline README.md:58]: Is the given link to the "getting started" documentation sufficient or do we want this expanded?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -20,6 +20,8 @@ We use GitHub to manage reviews of pull requests.
 
 Should you wish to work on an issue, please claim it first by commenting on the GitHub issue that you want to work on. This is to prevent duplicated efforts from other contributors on the same issue.
 
+When your contribution is ready, please open a pull request using the [provided template](https://github.com/SAP/cloud-sdk-java/blob/main/.github/PULL_REQUEST_TEMPLATE.md).
```
Comment (CharlesDuboisSAP) [inline CONTRIBUTING.md:23]:
```suggestion
When your contribution is ready, please open a [pull request](https://github.com/SAP/cloud-sdk-java/compare).
```

File: README.md [hunk 4]
```diff
@@ -10,11 +10,32 @@
...
+7. [Code of Conduct](#code-of-conduct)
+8. [Licensing](#licensing)
+
+## About This Project
 
 The SAP Cloud SDK for Java makes it easy to build highly connected, resilient, multi tenant SaaS applications on the SAP Business Technology Platform (SAP BTP).
 It enables you to perform outbound requests using various protocols while only writing the business logic.
 
+## Release Notes
+- For an overview of the latest change, please refer to our [release notes.](https://sap.github.io/cloud-sdk/docs/java/v5/release-notes).
```
Comment (CharlesDuboisSAP) [inline README.md:34]:
```suggestion
- For an overview of the latest changes, please refer to our [release notes.](https://sap.github.io/cloud-sdk/docs/java/v5/release-notes).
```

**Review by CharlesDuboisSAP — CHANGES_REQUESTED** [review]
The build badge is broken 
image
And the maven central badge should point to v5

**Review by cschubertcs — APPROVED** [review]

**Pull request merged**